iceoryx_posh/source/popo/rpc_header.cpp

```diff
@@ -83,6 +84,16 @@ bool RequestHeader::isFireAndForget() const noexcept
     return m_isFireAndForget;
 }
 
+RequestHeader* RequestHeader::fromPayload(void* const payload) noexcept
+{
+    return static_cast<RequestHeader*>(mepoo::ChunkHeader::fromUserPayload(payload)->userHeader());
```

Comment:
Segfault possibility when `payload == nullptr`.

Reply:
@elfenpiff Good point. `fromUserPayload()` checks this. But returns nullptr and then we do `->userHeader()`